src/index.html

```diff
@@ -19,9 +19,400 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
The link to Google Fonts is duplicated. You only need to include it once in your HTML file. Consider removing the duplicate link to optimize loading times.